### PR TITLE
Extend documentation for lib/partition_setup.pm

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/57917

### CODE REVIEW

File: lib/partition_setup.pm [hunk 1]
```diff
@@ -209,7 +210,7 @@ sub set_partition_size {
 
  resize_partition();
 
-Method assumes that correct disk is already selected
+Method assumes that correct disk is already selected.
```
Comment (asdil12) [inline lib/partition_setup.pm]:
```suggestion
Method assumes that correct disk is already selected.
```

File: lib/partition_setup.pm [hunk 2]
```diff
@@ -396,9 +400,12 @@ sub addlv {
 
 =head2 addboot
 
- addboot();
+ addboot($part_size, [%default_boot_sizes]);
```
Comment (asdil12) [inline lib/partition_setup.pm]:
```suggestion
 addboot($part_size);
```

`%default_boot_sizes` is not a parameter.

File: lib/partition_setup.pm [hunk 3]
```diff
@@ -396,9 +400,12 @@ sub addlv {
...
+
+Add a boot partition based on architecture.
 
-Adds a boot partition, calls C<addpart> for this
+C<$part_size> is the size of partition. 
+C<%default_boot_sizes> is pre-defined default boot size.
```
Comment (asdil12) [inline lib/partition_setup.pm]:
```suggestion
```

File: lib/partition_setup.pm [hunk 4]
```diff
@@ -568,9 +576,14 @@ sub take_first_disk_storage_ng {
...
- take_first_disk([iscsi => $iscsi]);
+ take_first_disk([%args]);
 
-Selects the first disk in the list to be partitioned
+Take the first disk to be partitioned. Take first storage ng if it is C<is_storage_ng>. 
+C<[%args]> is device type.
+
+Example:
+ 
+ take_first_disk iscsi => 1;
```
Comment (asdil12) [inline lib/partition_setup.pm]:
```suggestion
 take_first_disk(iscsi => 1);
```

File: lib/partition_setup.pm [hunk 5]
```diff
@@ -568,9 +576,14 @@ sub take_first_disk_storage_ng {
 
 =head2 take_first_disk
 
- take_first_disk([iscsi => $iscsi]);
+ take_first_disk([%args]);
 
-Selects the first disk in the list to be partitioned
+Take the first disk to be partitioned. Take first storage ng if it is C<is_storage_ng>.
```
Comment (asdil12) [inline lib/partition_setup.pm]: Could you rephrase that?
I don't really understand what you want to say in the 2nd sentence.

**Comment by Zaoliang** [comment]
updated now

File: lib/partition_setup.pm [hunk 6]
```diff
@@ -68,8 +68,8 @@ sub is_storage_ng_newui {
 
  wipe_existing_pastitions_storage_ng();
 
-Deletes all existing partitions in the expert partitioner
-Despite the name it does not check if it is run on a storage ng system
+Deletes all existing partitions in the expert partitioner.
+Despite the name it does not check if it is run on a storage ng system,
```
Comment (asdil12) [inline lib/partition_setup.pm]:
```suggestion
Despite the name it does not check if it is run on a storage ng system,
```

File: lib/partition_setup.pm (hunk 3 above)
Comment (asdil12) [reply lib/partition_setup.pm]: Is this still there?
`%default_boot_sizes` is not a parameter.

File: lib/partition_setup.pm (hunk 6 above)
Comment (asdil12) [reply lib/partition_setup.pm]: trailing space

Comment (Zaoliang) [reply lib/partition_setup.pm]: removed now
